courses/factories.py

```diff
@@ -35,6 +36,15 @@ class Meta:
         model = Company
 
 
+class PartnerFactory(DjangoModelFactory):
+    """Factory for Company"""
+
+    name = factory.Faker("partner")
```

Comment:
Once we make the partner name unique, This will also be changed to have a prepending dynamic/random name.